Apps/Common/MultiPatchModelGenerator.C

```diff
@@ -469,10 +471,19 @@ bool MultiPatchModelGenerator2D::createTopologySets (SIMinput& sim) const
     insertion(e1, e1f, TopItem(i*nx+1,1,1));
     insertion(e2, e2f, TopItem((i+1)*nx,2,1));
   }
+  for (const auto& e : e1)
+    xb.insert(e);
+  for (const auto& e : e2)
+    xb.insert(e);
```

Comment:
All the `e` objects here will be a `TopEntity` reference, so I rather would use `for (const TopEntity& e : e1)`, etc. here, and the reader will save a few seconds figuring out that himself ;)

Reply:
they are Topitem, but sure.